homeassistant/components/reolink/switch.py

```diff
@@ -276,6 +279,31 @@ async def async_setup_entry(
             if entity_description.supported(reolink_data.host.api)
         ]
     )
+
+    # Can be removed in HA 2025.2.0
+    entity_reg = er.async_get(hass)
+    reg_entities = er.async_entries_for_config_entry(entity_reg, config_entry.entry_id)
+    for entity in reg_entities:
+        if entity.domain == "switch" and entity.unique_id.endswith("_hdr"):
+            if entity.disabled:
+                entity_reg.async_remove(entity.entity_id)
+                continue
+
+            ir.async_create_issue(
```

Comment:
We shouldn't create an issue just because the entity is enabled. We don't know that the user is using the entity for anything. Praxis is to create an issue if an entity service is called.

Reply:
I will keep this in mind for next time.
However in this case this entity is disabled by default, so when a user enables it, I think it is safe to assume the enity is also beein used.

Reply:
Maybe they activated it and forgot about it?